frontend/ui/screensaver.lua

```diff
@@ -120,20 +133,6 @@ local function expandSpecial(message, fallback)
             authors = docinfo.data.doc_props.authors and docinfo.data.doc_props.authors ~= "" and docinfo.data.doc_props.authors or authors
             series = docinfo.data.doc_props.series and docinfo.data.doc_props.series ~= "" and docinfo.data.doc_props.series or series
         end
-        docinfo:close()
```

Comment:
That deletion is on purpose: DocSettings doesn't keep the file open, and close is a synonym for flush (i.e., dump the settings to the file).

Here, we're only reading, so that's completely unnecessary.